drivers/modem/modem_iface_uart.c

```diff
@@ -228,3 +228,25 @@ int modem_iface_uart_init(struct modem_iface *iface,
 
 	return 0;
 }
+
+void modem_iface_sleep(struct modem_iface *iface)
+{
+	uart_irq_rx_disable(iface->dev);
+#ifdef DEVICE_PM_LOW_POWER_STATE
```

Comment:
Better would be
```
if (IS_ENABLED(DEVICE_PM_LOW_POWER_STATE)) {
...
}
```
so that compiler will see the statement. Same comment to `modem_iface_wake()`

Reply:
In fact it's raising build issue as `device_set_power_state()`, `DEVICE_PM_LOW_POWER_STATE` and `DEVICE_PM_LOW_POWER_STATE` are not defined.

I therefore roll back from:
`if (IS_ENABLED(CONFIG_DEVICE_POWER_MANAGEMENT)) {`
to:
`#ifdef CONFIG_DEVICE_POWER_MANAGEMENT`